### ISSUE TITLE
Experiment Setting Mismatch

### ISSUE THREAD

**Issue by xingyul3** [aravindr93/hand_dapg#10]
Hi, thanks for releasing the code and demos. I would like to re-produce the results in your RSS2018 paper based on the released material. However, I found there are many mismatches between the description in the paper and the code in terms of detailed experiment settings.

First, in your RSS2018 paper, you mentioned you tested DAPG and BC init + NPG using a sparse reward setting. However, in the envs release [here](https://github.com/vikashplus/mj_envs/tree/master/mj_envs/hand_manipulation_suite), the rewards are not sparse but "shaped". Could you provide details on how you define the sparse reward for these environments? Or if possible, could you release the code of the sparse reward setting?

Second, [behavior cloning code](https://github.com/aravindr93/mjrl/blob/master/mjrl/algos/behavior_cloning.py) is not used in any mjrl algorithms including DAPG (thanks for releasing BC code though). Could you provide details on how it is used in DAPG during pre-training? Or if possible, could you release the code for BC from which the results in your RSS 2018 paper can be reproduced?

Third, [this script](https://github.com/aravindr93/mjrl/blob/master/examples/policy_opt_job_script.py) does not include DAPG. After I modified the code to include it, I found the algorithm cannot learn. I guess it's because I didn't do pre-training. Is that one possible reason?
